kafka/adminapi.go

```diff
@@ -560,6 +570,10 @@ type DeleteACLsResult = DescribeACLsResult
 // The returned result event is checked for errors its error is returned if set.
 func (a *AdminClient) waitResult(ctx context.Context, cQueue *C.rd_kafka_queue_t, cEventType C.rd_kafka_event_type_t) (rkev *C.rd_kafka_event_t, err error) {
 
+	if a.verifyAdminClient() == false {
+		return nil, newErrorFromString(ErrState, "not a valid adminClient state.")
```

Comment:
Instead of repeating this in all uses, suggest having verifyAdminClient return this error.
And perhaps also rephrase it, the user won't know what a valid or invalid adminClient state is, perhaps say:
"Operation not allowed on closed client"